lib/gui/app.js

```diff
@@ -176,6 +176,11 @@ app.run(() => {
     ].join(' '));
 
     windowProgress.set(currentFlashState.percentage);
+
+    const DASH_UNICODE_CHAR = '\u2013';
+    const flashProgress = messages.info.flashProgress(currentFlashState);
+    const title = `${packageJSON.displayName} ${DASH_UNICODE_CHAR} ${flashProgress}`;
+    windowProgress.setTitle(title);
```

Comment:
> Also, we should make this window progress function accept a "progress state" object (the one coming from etcher-image-write, which has the percentage, type, speed, etc) rather than just a percentage

Yeah, that's exactly what I was thinking with my `EDIT:` comment, but forgot to explicitly mention it :+1: